### ISSUE TITLE
source available, boooooo

### ISSUE THREAD

**Issue by thepwrtank18** [WinampDesktop/winamp#31]
this isn't open source

**Comment by Ryder17z** [comment]
it is a good laugh however sine they leaked the shoutcast3 source with this repo.

**Comment by Aerocatia** [comment]
They should just make it GPL, literally nothing will happen that can't happen already.

**Comment by repomansez** [comment]
and a "COPYLEFT" license that restricts freedoms

how can these people be even dumber than me? I'll never understand it

**Comment by kbity** [comment]
Copyleft DOES restrict freedom. This in inherent to Copyleft. No restrictions would be MIT. Copyleft however does NOT restrict freedom for... making forks...

**Comment by robvdl** [comment]
Can we just pool together and create a single fork and get the Linux port under way.

The community can work much faster anyway, I don't expect many changes to happen to the desktop client from their end.

**Comment by 0x5066** [comment]
> Can we just pool together and create a single fork and get the Linux port under way.
> 
> The community can work much faster anyway, I don't expect many changes to happen to the desktop client from their end.

How are you gonna have third party plugin support? All of the existing 3rd party plugins will not work anymore.
The only things you could derive from this is the Classic Skin engine and that's it

**Comment by robvdl** [comment]
I see, so that makes this a pretty crap release. Isn't there even the player engine?

**Comment by 0x5066** [comment]
As far as I can tell it has everything, the player core as well as the APIs.
What I meant with my comment was, all existing third party plugins are bound to the Windows API, so porting Winamp to Linux you'd lose all the Windows-only third party plugins.
"The only things you could derive from this is the Classic Skin engine and that's it" means that the only thing worthwhile from trying to port Winamp to Linux would be the Classic Skin engine.

**Comment by sudomgamal** [comment]
Lets port the skins to VLC or something. It's only about the skins right?

**Comment by robvdl** [comment]
No there is a lot more to it than just skins.

But since the license is so restrictive and no forks are allowed it's a pretty deadbeat release and I can't see myself contributing.

**Comment by 0x5066** [comment]
> Lets port the skins to VLC or something. It's only about the skins right?

VLC's skin engine probably does not even come close to winamp's own quirks, though yes it is more than just the skins

You can create your own Winamp-like player with what's available of the Winamp SDK, getting the classic skin engine to function as it does in Winamp is trickier, and the only program that's managed this is WACUP

**Comment by Ryder17z** [comment]
I'd pick "something else" over VLC, as VLC has decade-old problems,
especially with skins not getting fixed.

**Comment by notpushkin** [comment]
> "The only things you could derive from this is the Classic Skin engine and that's it" means that the only thing worthwhile from trying to port Winamp to Linux would be the Classic Skin engine.

There’s [Audacious](https://audacious-media-player.org/) already which is open source and has pretty good Classic Skin support. I wouldn’t say it’s 100% but for all practical purposes this is a solved problem.

**Comment by 0x5066** [comment]
> > "The only things you could derive from this is the Classic Skin engine and that's it" means that the only thing worthwhile from trying to port Winamp to Linux would be the Classic Skin engine.
> 
> There’s [Audacious](https://audacious-media-player.org/) already which is open source and has pretty good Classic Skin support. I wouldn’t say it’s 100% but for all practical purposes this is a solved problem.
> 
> (Also, [WebAmp](https://webamp.org/)!)

I'm more likely to vouch for Webamp than Audacious, Audacious' own Classic Skin support is not exactly the best and [an attempt at fixing it resulted in various fixed UI aspects being "user hostile".](https://github.com/audacious-media-player/audacious-plugins/pull/160)

**Comment by notpushkin** [comment]
@0x5066 Agreed, I too don’t think Audacious is the best implementation out there. Perhaps porting Webamp to a desktop would be feasible? (Well, there’s the [Electron port](https://github.com/durasj/webamp-desktop) already, but that’s a bit too heavy for a Winamp-like player IMO.)

**Comment by 0x5066** [comment]
One could theoretically take Audacious' skins plugin and build a new player from that codebase, something I probably should've done with mpdamp.
